src/NodaTime/TimeZones/DateTimeZoneCache.cs

```diff
@@ -80,15 +80,10 @@ public DateTimeZoneCache([NotNull] IDateTimeZoneSource source)
         /// <inheritdoc />
         public DateTimeZone GetSystemDefault()
         {
-            TimeZoneInfo bcl = TimeZoneInfo.Local;
-            string id = source.MapTimeZoneId(bcl);
+            string id = source.GetSystemDefaultId();
             if (id == null)
             {
-#if PCL
-                throw new DateTimeZoneNotFoundException("TimeZoneInfo name " + bcl.StandardName + " is unknown to source " + VersionId);
-#else
-                throw new DateTimeZoneNotFoundException("TimeZoneInfo ID " + bcl.Id + " is unknown to source " + VersionId);
-#endif
+                throw new DateTimeZoneNotFoundException("TimeZoneInfo ID " + id + " is unknown to source " + VersionId);
```

Comment:
Whoops, yes. Have also changed the source in this file to use interpolated string literals.